examples/log_parsing/preprocessing.py

```diff
@@ -80,7 +79,6 @@ def __init__(self,
         self._do_lower_case = do_lower_case
         self._add_special_tokens = add_special_tokens
 
-        self._tokenizer: SubwordTokenizer = None
```

Comment:
How are we doing the tokenization now, or did we just move this to a different location?

Reply:
Disregard, I saw your comment above.